hip-more-cultural-names/common/landed_titles/0_HIP_MoreCulturalNames.txt

```diff
@@ -7707,11 +8276,16 @@ e_hre = {
                     roman = "Glacio"
                 }
                 b_jicin = {
+                    bosnian = "Jic�n" # Historical? Copied from Croatian
```

Comment:
if this is í its false this letter doesnt exist in south slavic alphabet

Reply:
i'm currently experimenting, i have to do a lot of cleaning still